### PR TITLE
corrected track_total_hits docs

### PR DESCRIPTION
**Pull request by JnBrymn**
Here's how `track_total_hits` behaves for me.
```
$ curl "https://my_es_cluster.wherever.net/standard_production_v1-0000004/_search?timeout=250ms"  | jq .hits.total.value
10000

$ curl "https://my_es_cluster.wherever.net/standard_production_v1-0000004/_search?timeout=250ms&track_total_hits=true"  | jq .hits.total.value
384255594
```

### CODE REVIEW

**Comment by JnBrymn** [comment]
I just signed the committer agreement. Now immortalize me by making me an Elasticsearch contributor. _I am ready._

**Comment by jrodewig** [comment]
@elasticmachine test this please

File: docs/reference/search/search.asciidoc [hunk 1]
```diff
@@ -315,8 +315,8 @@ the scores are not used for sorting. Defaults to `false`.
 (Optional, integer or boolean)
 Number of hits matching the query to count accurately. Defaults to `10000`.
 +
-If `true`, the exact number of hits is returned. If `false`, the response does not
-include the total number of hits matching the query.
+If `true`, exact number of hits is returned at the cost of some performance.
```
Comment (jrodewig) [inline docs/reference/search/search.asciidoc:318]:
```suggestion
If `true`, the exact number of hits is returned at the cost of some performance.
```

**Comment by jrodewig** [comment]
@elasticmachine test this please

**Comment by jrodewig** [comment]
Hi @JnBrymn!

Thanks for opening this PR. Before we immortalize you, can sign the CLA using the email address associated with your commit?

You can find that email address here:
https://github.com/elastic/elasticsearch/commit/acaa5b10b15f47ed9935d5865a7fcbb06c08219c.patch

If the CLA was signed with a different email address, our bot won't recognize it.

Thanks again for the contribution and going through the extra bit of ceremony.

**Comment by JnBrymn** [comment]
Signed again!

**Comment by jrodewig** [comment]
@elasticmachine test this please

**Pull request merged**